v3/internal/runtime/desktop/@wailsio/runtime/src/dialogs.ts

```
@@ -0,0 +1,255 @@
+ * @param options - Dialog options.
+ * @returns A promise that resolves with the label of the chosen button.
+ */
+export function Warning(options: MessageDialogOptions): Promise<string> { return dialog(DialogWarning, options); }
+
+/**
+ * Presents an error dialog.
+ *
+ * @param options - Dialog options.
+ * @returns A promise that resolves with the label of the chosen button.
+ */
+export function Error(options: MessageDialogOptions): Promise<string> { return dialog(DialogError, options); }
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)